BibBuddy/app/build.gradle

```diff
@@ -13,7 +16,7 @@ android {
 
     defaultConfig {
         applicationId "de.bibbuddy"
-        minSdkVersion 24
+        minSdkVersion 26
```

Comment:
Why not support Android 7 anymore?

Reply:
Android Studio told me to change it to 26 since we use an Android element that is not supported below and might cause problems. We talked about it in person already though. I changed it back to 24 for our device testing for now.